native/packages/react-native-bpk-component-carousel-indicator/stories.js

```diff
@@ -128,7 +130,8 @@ class StatefulBpkCarouselIndicatorExample extends React.Component<
           <ScrollView
             horizontal
             pagingEnabled
-            onMomentumScrollEnd={this.handleScroll}
+            onScroll={this.handleScroll}
+            scrollEventThrottle={16}
```

Comment:
I'm thinking actually maybe we could edit this story to not include a makeshift carousel, now that we have `BpkCarousel` we don't need to duplicate it here. What do you reckon?

Reply:
I think that's a good idea, but I also think we need one story to show the animation, it can be only the dots without the scroll view. What do you think?

Reply:
That sounds good. I've updated the story to do that:

<img width="487" alt="screen shot 2018-06-06 at 09 44 40" src="https://user-images.githubusercontent.com/73652/41027163-3eaed7a6-696e-11e8-91f5-0175c05d7129.png">